src/backends/tensorrt/tensorrtlib.cc

```diff
@@ -603,6 +621,24 @@ namespace dd
                 throw MLLibBadParamException(
                     "timeseries not yet implemented over tensorRT backend");
               }
+            // GAN / raw output
+            else if (!extract_layer.empty())
+              {
+                _buffers.resize(2);
+                _floatOut.resize(_max_batch_size * _dims.d[1] * _dims.d[2]
+                                 * _dims.d[3]);
```

Comment:
check that _dims.nbDims >= 4?

Reply:
Sure, I'll add this. Rational being that an image has channels, height, width and batch size.